sections/semantics-sections.include

```diff
@@ -1255,10 +1255,10 @@
   The <code>h1</code>–<code>h6</code> elements are headings.
 
   The first element of <a>heading content</a> in an element of <a>sectioning content</a>
-  <a>represents</a> the heading for that (explicit) section. Subsequent headings of equal or higher
-  <a>rank</a> start new (implied) subsections that are part of the previous section's parent section.
-  Subsequent headings of lower <a>rank</a> start new (implied) subsections that are part of the previous
-  one. In both cases, the element <a>represents</a> the heading of the implied section.
+  <a>represents</a> the heading for that explicit section. Subsequent headings of equal or higher <a>rank</a>
+  start new implied subsections that are part of the previous section's parent section. Subsequent
+  headings of lower <a>rank</a> start new implied subsections that are part of the previous one.
```

Comment:
I thought this text (even before you edited it) was *wrong*, and subsequent headings of lower rank start implied subsections of the **current** section, not the previous one.

It's actually just horribly ambiguous. It should be clear whether it means "a subsequent heading of lower rank in the same sectioning element" - which starts a new subsection in the current section - or "a subsequent heading, in its own sectioning element, after an implied section" - which I *think* starts a new subsection of the previous section's parent in the same way a subsequent equal or higher-ranked heading in the same subsection does.

My review suggestion is that we merge your changes with thanks, and note, as if we didn't already know it, that this section is still messy.